src/main/java/seedu/algobase/ui/MainWindow.java

```diff
@@ -134,6 +132,18 @@ void fillInnerParts() {
         commandBoxPlaceholder.getChildren().add(commandBox.getRoot());
     }
 
+    private DisplayTabPane getDisplayTabPane() {
+        problemListPanel = new ProblemListPanel(logic.getProcessedProblemList());
+        planListPanel = new PlanListPanel(logic.getProcessedPlanList());
+        taskListPanel = new TaskListPanel(logic.getProcessedTaskList());
+        DisplayTab problemListPanelTab = new DisplayTab(ModelEnum.PROBLEM.getTabName(), problemListPanel);
+        DisplayTab tagListPanelTab = new DisplayTab(ModelEnum.TAG.getTabName());
```

Comment:
Is this because Tag still lack a functional GUI?

Reply:
Yup